demo/demo-spring-boot-discovery/demo-spring-boot-discovery-client/src/test/java/io/servicecomb/demo/discovery/client/DiscoveryClientIT.java

```diff
@@ -0,0 +1,24 @@
+package io.servicecomb.demo.discovery.client;
+
+import org.junit.Before;
+import org.junit.Test;
+import io.servicecomb.demo.TestMgr;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class DiscoveryClientIT {
+
+	@Before
+	public void setUp() {
+		TestMgr.errors().clear();
+	}
+
+	@Test
```

Comment:
this test doesn't have much meaning except ensured DiscoveryClient can be started up.

Nothing related to how services discovery with spring cloud integration and load balancing was tested.

We have to be sure we are able to forward requests to corresponding services from zuul with service center